tools/docker-compose/start_development.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set +x
 
-if [ `id -u` -ge 500 ]; then
+if [ `id -u` -ge 500 ] || [ -z "${CURRENT_UID}" ]; then
```

Comment:
This whole block actually needs to go in `/bootstrap_development.sh`... I'll run that command standalone sometimes but it's *always* called from `/start_development.sh`